web/vital_records/templates/vital_records/_button_row.html

```diff
@@ -0,0 +1,6 @@
+<div class="btn-row">
+    <a class="btn btn-outline-primary"
+       role="button"
+       href="{{ previous_url }}">Back</a>
```

Comment:
Originally, the `href` line was being set with: `{% url previous_route pk=object.id %}` - a URL being created from the supplied `previous_route` from the context, and the ID key being passed in.

This was causing a bug on the Type form view, where the `previous_route` is going back to `request_start`, which does not need the ID in the URL. 

Instead, I changed the includes HTML to be as simple as possible, by directly passing in the _full URL, with our without the ID kwargs_, instead of the caluclating the `url` from the `route` in the template. I added a new context key/value, `previous_url`, which is now being set in `views.py`.

Reply:
That makes sense, nice fix!

For the views that are inheriting from `StepsMixin`, can we move
```
context["previous_url"] = reverse(context["previous_route"], kwargs={"pk": self.object.pk})
```
(e.g. in [`NameView`](https://github.com/Office-of-Digital-Services/cdt-ods-disaster-recovery/pull/319/files#diff-5f9001db61faca1d31575b962487dec922eae2664a66f78cfb1e0f8695a8e929R139)) into the mixin?